slamon_afm/tests/models_tests.py

```diff
@@ -0,0 +1,122 @@
+from datetime import datetime
+
+from slamon_afm.models import db, Task, Agent, AgentCapability
+from slamon_afm.tests.afm_test import AFMTest
+
+
+class TaskUnassignTests(AFMTest):
+    """
+    These tests are not actually testing any code but continuing
+    the tradition of using DB withing unittests by checking if the
+    SQLAlchemy declarations work as intended.
+    """
+
+    def setUp(self):
+        super(TaskUnassignTests, self).setUp()
+
+        self.agent = Agent(
+            uuid='de305d54-75b4-431b-adb2-eb6b9e546013',
+            name='test agent',
+            last_seen=datetime.utcnow()
+        )
+        db.session.add(self.agent)
+
+        self.task = Task(
+            uuid='de305d54-75b4-431b-adb2-eb6b9e546013',
+            test_id='de305d54-75b4-431b-adb2-eb6b9e546013',
+            type='task-type-1',
+            version=1,
+            data="{}",
+            assigned_agent=self.agent
+        )
+        db.session.add(self.task)
+
+        db.session.flush()
```

Comment:
It might be worth asserting something like:

```python
self.assertEqual(len(self.agent.tasks), 1)
self.assertIsNotNone(self.task.assigned_agent_uuid)
```
Since in the tests you verify that the counts have dropped to zero, but you never actually verify that they were added in the first place.